### PR TITLE
code examples and some styling

### PR DESCRIPTION
**Pull request by MrTim**
Hey,
Here are some changes for third lecture. Purpose of the lecture is to introduce some basics and start writing some code.

Please go through the lecture and leave any comments.
Homework would be done in separate repo I guess.

### CODE REVIEW

File: lessons/3_first_interactive_component/index.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
 	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/marked.js">
 	<link rel="JavaScript" href="../../reveal.js/plugin/markdown/markdown.js">
 	<style>
+		b {
+			background-color: #ad0170;
```
Comment (MrTim) [inline lessons/3_first_interactive_component/index.html:12]: for some reason after pushing those weird spaces appear...need to check it.

File: lessons/3_first_interactive_component/index.html [hunk 2]
```diff
@@ -364,29 +364,92 @@ <h3>For that you need to do:</h3>
...
+			</section>
+			<section>
+				<p>We will use <b>&lt;script/&gt;</b> tag inside <b>index.html</b></p>
+				<pre>
+					<code>
+&lt;html&gt;
+ &lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+	&lt;title&gt;My HTML Page&lt;/title&gt;
+	&lt;script type="text/javascript"&gt;
```
Comment (caritosteph) [inline lessons/3_first_interactive_component/index.html]: I think **script** should be in the **body** tag like you are using in the example.

Comment (MrTim) [reply lessons/3_first_interactive_component/index.html]: Done!

**Review by caritosteph — APPROVED** [review]

File: lessons/3_first_interactive_component/goodBadBtnExample.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      background-color: lightgreen;
+    }
+    #badButton {
+      padding: 0.5em 2em;
+      font-size: 20pt;
+      background-color: lightcoral;
+    }
+  </style>
+  <script>
+    let goodButtonCmp = document.getElementById("goodButton");
```
Comment (BTBTravis) [inline lessons/3_first_interactive_component/goodBadBtnExample.html:32]: I'm torn on `let` vs `const` here for the absolute beginner. `let` sounds kinda better when pronounced as an English sentence, "Let GoodButtonCmp equal..." but `const` may help them avoid reassignment mistakes

File: lessons/3_first_interactive_component/index.html [hunk 4]
```diff
@@ -331,29 +364,92 @@ <h3>For that you need to do:</h3>
...
 				</ul>
 			</section>
+			<section>
+				<h3>But where do we write JavaScipt code?</h3>
+			</section>
+			<section>
+				<p>We will use <b>&lt;script/&gt;</b> tag inside <b>index.html</b></p>
+				<pre>
+					<code>
+&lt;html&gt;
```
Comment (BTBTravis) [inline lessons/3_first_interactive_component/index.html:374]: Christ. did you have to hand type this or did you use encode tool?

File: lessons/3_first_interactive_component/index.html [hunk 5]
```diff
@@ -331,29 +364,92 @@ <h3>For that you need to do:</h3>
...
         </ul>
-      </section>
+			</section>
+			<section>
+				<h3>Practice what we learned in the class (PART 2)</h3>
+				<p>For main project (current index.html)</p>
+				<ul>
+          <li>Make the burger icon react onClick</li>
+          <li>If user clicked it, show an alert('something') or console.log('something')</li>
+          <li>Don't forget to push to github!</li>
```
Comment (BTBTravis) [inline lessons/3_first_interactive_component/index.html]: maybe we can link to @caritosteph 's git homework submitting instructions in #11

File: lessons/3_first_interactive_component/index.html [hunk 6]
```diff
@@ -331,29 +364,92 @@ <h3>For that you need to do:</h3>
...
-					<li>Create a button</li>
-					<li>Make this button do the console.log('Clicked') if user clicked it</li>
+					<li>Copy your index.html file and name it Lesson3.html</li>
+					<li>Create a Feel good button </li>
+					<li>Make this button do the console.log('I feel really good today') if user clicked it</li>
 					<li>HINT: for that you need to apply all what we just learned :)</li>
 				</ul>
 			</section>
 			<section>
 				<h1>Homework</h1>
```
Comment (BTBTravis) [inline lessons/3_first_interactive_component/index.html:436]: I think we should move all homework to markdown files that clean easy to reference place for the students like we did for homework one https://mrtim.github.io/js-munich-2020-spring/homework/1_base_page

**Review by BTBTravis — COMMENTED** [review]
@MrTim left some comments nothing to critical

File: lessons/3_first_interactive_component/goodBadBtnExample.html (hunk 3 above)
Comment (MrTim) [reply lessons/3_first_interactive_component/goodBadBtnExample.html:32]: Hm, I still would stick with let for now, just to make it easier. When we go further we can use proper const and let when they needed.

File: lessons/3_first_interactive_component/index.html (hunk 4 above)
Comment (MrTim) [reply lessons/3_first_interactive_component/index.html:374]: nope, I did copy most of it stuff from other slides :)

File: lessons/3_first_interactive_component/index.html (hunk 6 above)
Comment (MrTim) [reply lessons/3_first_interactive_component/index.html:436]: Normally there is a separate slides set for homework, like here https://mrtim.github.io/js-munich-2020-spring/lessons/2_git_time/homework#/3
I just forgot to make the same thing here.

**Pull request merged**